time stamp

float64 cartographer_x
float64 cartographer_y
float64 cartographer_th
float64[9] cartographer_cov

float64 gps_x
float64 gps_y
float64 gps_th
float64[9] gps_cov

float64 dragonfly_x
float64 dragonfly_y
float64 dragonfly_th
float64[9] dragonfly_cov

float64 deadr_x
float64 deadr_y
float64 deadr_th
float64[9] deadr_cov

float64[6] dij